keras_cv/layers/preprocessing/random_contrast.py

```diff
@@ -56,33 +55,44 @@ class RandomContrast(BaseImageAugmentationLayer):
 
     def __init__(self, factor, seed=None, **kwargs):
         super().__init__(seed=seed, force_generator=True, **kwargs)
+        self.factor = factor
         if isinstance(factor, (tuple, list)):
-            min = 1 - factor[0]
-            max = 1 + factor[1]
+            self.lower = factor[0]
+            self.upper = factor[1]
         else:
-            min = 1 - factor
-            max = 1 + factor
+            self.lower = self.upper = factor
+        if self.lower < 0.0 or self.upper < 0.0 or self.lower > 1.0:
+            raise ValueError(
+                "`factor` argument cannot have negative values or values "
+                "greater than 1."
+                f"Received: factor={factor}"
+            )
         self.factor_input = factor
-        self.factor = preprocessing.parse_factor((min, max), min_value=-1, max_value=2)
         self.seed = seed
 
-    def get_random_transformation(self, **kwargs):
-        return self.factor()
-
-    def augment_image(self, image, transformation, **kwargs):
-        contrast_factor = transformation
-        output = tf.image.adjust_contrast(image, contrast_factor=contrast_factor)
+    def get_random_transformation_batch(self, batch_size, **kwargs):
+        return
+
+    def augment_images(self, images, transformations, **kwargs):
+        # Note: we skip get_random_transformation_batch() in RandomContrast because the
```

Comment:
This comment seems like it belongs in the `get_random_transformation_batch` impl